client/src/services/link.tsx

```diff
@@ -38,7 +38,7 @@ type LinkAction =
 
 interface LinkContextShape extends LinkState {
   dispatch: Dispatch<LinkAction>;
-  generateLinkToken: (userId: number, transferIntentId: string) => string;
+  generateLinkToken: (userId: number, subscriptionAmount: number) => string;
```

Comment:
For a future refactor, you might want to rename this something like generateLinkTokenForTransfer() just to make it clear we're doing some custom transfer-only work here

Reply:
done